specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```
@@ -0,0 +1,2239 @@
+        }
+      },
+      "type": "object"
+    },
+    "ConversationTasksState": {
+      "description": "Contains the state for the tasks being executed as part of the analyze conversation job submitted.",
+      "properties": {
+        "tasks": {
+          "properties": {
+            "completed": {
+              "description": "Count of tasks completed successfully.",
+              "type": "integer"
+            },
+            "failed": {
+              "description": "Count of tasks that failed.",
+              "type": "integer"
+            },
+            "inProgress": {
+              "description": "Count of tasks in progress currently.",
+              "type": "integer"
+            },
+            "total": {
+              "description": "Total count of tasks submitted as part of the job.",
+              "type": "integer"
+            },
+            "items": {
+              "description": "List of results from tasks (if available).",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AnalyzeConversationResult"
+              }
+            }
+          },
+          "required": [
+            "total",
+            "completed",
+            "failed",
+            "inProgress"
+          ],
+          "type": "object"
+        }
+      },
+      "required": [
+        "tasks"
+      ],
+      "type": "object"
+    },
+    "AnalyzeConversationJobResult": {
+      "type": "object",
+      "description": "Container for results for all tasks in the conversation job.",
+      "discriminator": "kind",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationResultsKind"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/TaskState"
+        },
+        {
+          "$ref": "common.json#/definitions/TaskIdentifier"
+        }
+      ],
+      "required": [
+        "kind"
+      ]
+    },
+    "ConversationResultBase": {
+      "type": "object",
+      "description": "Shared attributes for all conversational task results.",
+      "required": [
+        "id",
+        "warnings"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique, non-empty conversation identifier."
+        },
+        "warnings": {
+          "type": "array",
+          "description": "Warnings encountered while processing document.",
+          "items": {
+            "$ref": "#/definitions/ConversationWarning"
+          }
+        },
+        "statistics": {
+          "$ref": "#/definitions/ConversationStatistics"
+        }
+      }
+    },
+    "ConversationError": {
+      "type": "object",
+      "description": "Contains details of errors encountered during a job execution.",
+      "required": [
+        "id",
+        "error"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The id of the conversation."
```

Nit: here and elsewhere (like with my comment about "Id" in descriptions):
```suggestion
          "description": "The ID of the conversation."
```